parsers/US_MISO.py

```diff
@@ -76,40 +84,39 @@ def data_processer(json_data, logger: Logger):
     tzinfos = {"EST": tz.gettz("America/New_York")}
     dt = parser.parse(time_data, tzinfos=tzinfos)
 
-    return dt, production
+    return dt, mix
 
 
 def fetch_production(
-    zone_key: str = "US-MISO",
+    zone_key: ZoneKey = ZoneKey(ZONE),
```

Comment:
Just a small note, you don't have to add a default value here. We usually do it for single zone parsers because it's easy to understand but it's not required by any means.